### PR TITLE
Add PAPER space

### PR DESCRIPTION
**Pull request by flushlol**

### CODE REVIEW

File: skins/flush.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.flush {
+  --primary-color: #7970bf;
+  --header-bg: #1a1721;
+  --bg-color: #241f2c radial-gradient(circle at 50% 0,#241f2c,#131018 99%);
+  --border-color: rgba(0, 0, 0,.5);
+  --text-color: rgba(255, 255, 255, .7);
+  --link-color: #FFF;
+  --heading-color: #FFF;
+}
```
Comment (gokaiorg) [inline skins/flush.scss:9]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Review by gokaiorg — APPROVED** [review]

File: skins/flush.scss (hunk 1 above)
Comment (flushlol) [reply skins/flush.scss:9]: Thank you for the review!
Fixed, please check out the last updates

**Review by gokaiorg — APPROVED** [review]

**Comment by flushlol** [comment]
@gokaiorg Hi there!
It's almost 5 hours since you merged our pr, but we cant see our skin in the setting, and "gov.flush.lol" redirects to snapshot main page

**Comment by gokaiorg** [comment]
@flushlol It's look good to me now.
image

**Pull request merged**